engine/common/host.c

```diff
@@ -862,7 +862,9 @@ void Host_FreeCommon( void )
 
 	Mem_FreePool( &host.mempool );
 }
-
+#ifdef PANDORA
+int noshouldermb = 0;
+#endif
 /*

```

Comment:
noshouldermb is Pandora specific, and alow to inibit the Shoulder Button as Mouse Button (Pandora soulders are Right Shift and Right Control keys).
I could have made cvar, like for the ToggleCrouch, but because it's very pandora specific, I haven't taken the time to do it properly.